cps/CP102.md

```diff
@@ -104,5 +104,5 @@ Preimage hash will be submitted as a motion by a Councilor. After Council vote,
 
 Link to the RFC on the Forum: https://gov.centrifuge.io/t/cp102-deposit-additional-cfg-into-hydradx-omnipool/6326/
 
-Link to the onchain vote: 
+Link to the onchain vote: https://centrifuge.subsquare.io/council/motions/92
```

Comment:
You don't want to link to the referendum vote instead?

https://centrifuge.subsquare.io/democracy/referenda/55